test/parallel/test-cluster-setup-master.js

```diff
@@ -58,37 +58,34 @@ if (cluster.isWorker) {
     }
   });
 
-  let correctIn = 0;
+  let correctInput = 0;
 
-  cluster.on('online', function lisenter(worker) {
-
-    onlineWorkers++;
+  cluster.on('online', common.mustCall(function listener(worker) {
 
     worker.once('message', function(data) {
-      correctIn += (data === 'custom argument' ? 1 : 0);
-      if (correctIn === totalWorkers) {
+      correctInput += (data === 'custom argument' ? 1 : 0);
+      if (correctInput === totalWorkers) {
         checks.args = true;
       }
       worker.kill();
     });
 
-    // All workers are online
-    if (onlineWorkers === totalWorkers) {
-      checks.workers = true;
-    }
-  });
+  }, totalWorkers));
 
   // Start all workers
   cluster.fork();
   cluster.fork();
 
   // Check all values
   process.once('exit', function() {
-    assert.ok(checks.workers, 'Not all workers went online');
-    assert.ok(checks.args, 'The arguments was noy send to the worker');
+    const argsMsg = `The arguments was not send for one or more worker.
+                       There was  ${correctInput} worker that receive
```

Comment:
The indentations will be preserved when they show up in assertions. Can you change this to using concatenations instead?